packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -60,24 +60,34 @@ let scheduleWork: (
   callback: FrameCallbackType,
   options?: {timeout: number},
 ) => number;
-let cancelScheduledWork: (callbackID: number) => void;
+let cancelScheduledWork: (callbackId: number) => void;
 
 if (!ExecutionEnvironment.canUseDOM) {
+  let callbackIdCounter = 0;
+  // Timeouts are objects in Node.
+  // For consistency, we'll use numbers in the public API anyway.
+  const timeoutIds: {[number]: TimeoutID} = {};
```

Comment:
The advantage of this isn't obvious to me.

Reply:
In Node `setTimeout` gives you an object, not a number.
This is a primarily Node code path.

Since our intentional API is to return a number I think it would be confusing if we started returning objects in Node environment.